airflow/configuration.py

```diff
@@ -372,6 +371,26 @@ def getsection(self, section):
             _section[key] = val
         return _section
 
+    def write(self, fp, space_around_delimiters=True):
+        """Write an .ini-format representation of the configuration state.
+
+        If `space_around_delimiters' is True (the default), delimiters
+        between keys and values are surrounded by spaces.
+
+        This is based on the configparsser.RawConfigParser.write method code to add support for
```

Comment:
This should probably be a # comment, not in the doc string?

What's changed? Is it just that the base one doesn't call `self.getsection(section)`?

Reply:
Does this make it run/evaluate the `_cmd` variants too? Do we want it to do that?

Reply:
```diff
    def write(self, fp, space_around_delimiters=True):
        """Write an .ini-format representation of the configuration state.

        If `space_around_delimiters' is True (the default), delimiters
        between keys and values are surrounded by spaces.
        """
        if space_around_delimiters:
            d = " {} ".format(self._delimiters[0])
        else:
            d = self._delimiters[0]
        if self._defaults:
            self._write_section(fp, self.default_section, self._defaults.items(), d)
        for section in self._sections:
-            self._write_section(fp, section, self._sections[section].items(), d)
+           self._write_section(fp, section, self.getsection(section).items(), d)
```

Reply:
We should also support the cmd variant. I added support.